lib/middleware.js

```diff
@@ -243,7 +243,7 @@ function checkImports(path, fn) {
 
 function compare(pathA, pathB) {
   pathA = pathA.split(sep);
```

Comment:
patha is the destination path.  this will be os dependent and we should split on `sep`.